catalog/ui/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.hbs

```diff
@@ -11,29 +11,20 @@
  *
  **/
  --}}
-<div class="interaction interaction-type" title="Change the form used to construct the search." data-help="Change the form used to construct the search.">
-</div>
-<div class="interaction interaction-type-text interaction-form" title="Change the form used to construct the search to the text form." data-help="Change the form used to construct the search to the text form.">
-    <div class="interaction-text">
-        <span class="fa fa-level-up"></span> Text
-    </div>
-</div>
-<div class="interaction interaction-type-basic interaction-form" title="Change the form used to construct the search to the text form." data-help="Change the form used to construct the search to the text form.">
+<div class="interaction interaction-type" title="Change the form used to construct the search." data-help="Change the form used to construct the search."></div>
+
+<div class="interaction interaction-type-advanced interaction-form" title="Change the form used to construct the search." data-help="Change the form used to construct the search.">
+    <div class="interaction-icon fa fa-question-circle"></div>
```

Comment:
nit: question mark seems a bit odd for "advanced" maybe something like `<i class="fas fa-search"></i>`

Reply:
yeah, seems more consistent with fa-search